src/api/channel/channel.ts

```diff
@@ -0,0 +1,157 @@
+type ChannelId = string;
+
+declare interface Listener{    
```

Comment:
We should reuse https://fdc3.finos.org/docs/1.0/api/DesktopAgent#listener for this, which is already ratified as part of v1.0 spec.